### ISSUE TITLE
WebDriver cannot start IE11 on Windows 7

### ISSUE THREAD

**Issue by mazswojejzony** [Polymer/wct-local#13]
Looks like the issue is related to the following function:
```
function internetExplorer(browser) {
  return {
    'browserName': 'ie',
    'version':     browser.version,
  };
}
```
Once I changed the `browserName` to `'internet explorer'` everything worked fine.

**Comment by mazswojejzony** [comment]
Any chance to fix this soon? I can prepare a PR if it helps.

**Comment by BorntraegerMarc** [comment]
Hi @mazswojejzony This actually worked! Thanks you very much :)

**Comment by BorntraegerMarc** [comment]
@mazswojejzony Could you do a PR for this and maybe release a new version?

**Comment by mazswojejzony** [comment]
I can prepare a PR, though I am not a repo owner so I cannot make a release. BTW: seems no one from Polymer team tracks wct-local issues. :-/

**Comment by BorntraegerMarc** [comment]
Hmm shame :/ Would be very glad, if you could do the PR :) Friendly ping to @notwaldorf and @addyosmani Would it be possible to release a new version? :)

**Comment by BorntraegerMarc** [comment]
Had some time, so I did the PR...

**Comment by mazswojejzony** [comment]
Great, thanks!

**Comment by taylor-smith** [comment]
This fix worked for me, though I'm on Windows 8.1

**Comment by azakus** [comment]
Should be fixed by #16
